Remove PRs to Endless Sky Assets and High DPI repos from the PR template (#165)
endless-sky/Endless-Sky-Delta: .github/pull_request_template.md

## Artwork Checklist
(If any artwork was added or changed by this PR, the following must be provided.)
 - [ ] I updated the copyright attributions, or decline to claim copyright of any assets produced or modified.
  - [ ] Include the '@2x' version of any added images alongside the regular image.

> 

Author: ```suggestion
```

## Artwork Checklist
(If any artwork was added or changed by this PR, the following must be provided.)

>  - [ ] I updated the copyright attributions, or decline to claim copyright of any assets produced or modified.

Author: ```suggestion
 - [ ] I updated the copyright attributions, or decline to claim copyright of any assets produced or modified.

Please upload a zipped file with @2x assets and HighDPI images to the main body of your pull request.
```

## Artwork Checklist
(If any artwork was added or changed by this PR, the following must be provided.)

>  - [ ] I updated the copyright attributions, or decline to claim copyright of any assets produced or modified.

Reviewer: ```suggestion
 - [ ] I updated the copyright attributions, or decline to claim copyright of any assets produced or modified.
 - [ ] Include the '@2x' version of any added images alongside the regular image.
```

>   - [ ] Include the '@2x' version of any added images alongside the regular image.

Reviewer: ```suggestion
 - [ ] Include the '@2x' version of any added images alongside the regular image.
```

I think I had an extra space in there in the original suggestion. Sorry

> Please upload a zipped file with @2x assets and HighDPI images to the main body of your pull request.

Reviewer: ```suggestion
- [ ] Please upload a zipped file with the necessary image, blend, and texture assets to the main body of your pull request.
```

> - [ ] Please upload a zipped file with the necessary image, blend, and texture assets to the main body of your pull request.

Reviewer: ```suggestion
 - [ ] Please upload a zipped file with the necessary image, blend, and texture assets to the main body of your pull request.
```

And I missed a space here.
